app/services/fdc_service.rb

```diff
@@ -0,0 +1,19 @@
+class FdcService
+
+  def search(food)
+    get_url("/foods/search?query=#{food}&pageSize=10")
+  end
+
+  def get_url(url)
+    response = conn.get(url)
+    JSON.parse(response.body, symbolize_names: true)
+  end
+
+  def conn
+    Faraday.new(url: "https://api.nal.usda.gov/fdc/v1/") do |f|
+      f.params['api_key'] = ''
```

Comment:
Your setup looks great so far, I like your PORO and Service methods. What you want here is 

```
Rails.application.credentials # then the way you access your key/value pair. 
```

If you forget what you called it in your setup, you can also do `rails c` in Terminal and type `Rails.application.credentials` so you can see your key/value.

Reply:
The command to set up your credentials initially is

```
EDITOR="code --wait" rails credentials:edit
```

I have this programmed as an alias in my terminal called `creds` - if you use aliases I strongly recommend adding this one, because the command is so wonky and easily forgettable.